zigpy_znp/tools/network_backup.py

```diff
@@ -84,7 +85,7 @@ async def backup_network(znp: ZNP) -> t.JSONType:
 
     now = datetime.datetime.now().astimezone()
 
-    obj["metadata"]["source"] = f"zigpy-znp@{importlib.metadata.version('zigpy-znp')}"
```

Comment:
Ok, we don't really need to replace this in the old CLI tools, but I guess it also doesn't hurt..